### ISSUE TITLE
Bad text highlight in dark mode

### ISSUE THREAD

**Issue by Nerrror** [arXiv/html_feedback#5072]
### Description

In dark mode, the main body background color is uncomfortable to read due to contrast and lines in the main text body

### (Optional:) Please add any files, screenshots, or other information here.

<img width="1701" height="1294" alt="Image" src="https://github.com/user-attachments/assets/a272f8d6-0f03-406b-a4ff-8120c450cd92" />

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

6daa87ea-b2de-4eb5-8e46-daf0041f5774

### Paper URL

https://arxiv.org/html/2412.04604v1

### Browser

Firefox/144.0

### Device Type

Desktop

**Comment by dginev** [comment]
Based on the crema/cream highlight you are showing, I am wondering if you arrived at that link from a web search result? 

Some browsers allow linking to a highlighted text fragment such as:
https://arxiv.org/html/2412.04604v1#S1:~:text=Alongside%20this%20definition,count%20of%20three)

And I can reproduce the crema color in Firefox today. That can be adjusted with some CSS:
```css
::target-text {
  background-color: ...;
}
```

But the question is what would be an appropriate color? @Nerrror do I understand correctly that you find the black/cream contrast hard to read?

**Comment by Nerrror** [comment]
Yes, I arrived from the author website "https://arcprize.org/arc-agi/1/"  which links to the arxiv report via this link in the main body: "[ARC Prize 2024 Technical Report](https://arxiv.org/html/2412.04604v1#:~:text=Fran%C3%A7ois%20Chollet%20first%20wrote%20about,19)."

when opening the html format directly from the abstract site, the formatting of the document seems to be correct.

Edit: maybe I mistook the "highlight" as a "formatting problem" because most of the report is highlighted which was unexpected

**Comment by Nerrror** [comment]
@dginev I think I mistook the "highlight" as a "formatting problem" because most of the report is highlighted which was unexpected - this seems to be correct behaviour and a problem on the side from the linking website rather a problem with the formatting
